integration-tests/tests/ubuntuFan_test.go

```diff
@@ -1,5 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
-// +build !excludeintegration
+// +build !excludeintegration,!excludeflaky
```

Comment:
The proposed feature looks fine, but is there a good reason to have these flaky tests in the first place?  In my experience flaky tests are consistently ignored because their failure isn't indicative of something relevant, which means they're much worse than having nothing at all: they teach ourselves to ignore actual errors. Oops.

My recommendation is to drop the flaky tests altogether. If they're still relevant, they may be moved into a branch and worked until they're not flaky anymore. If we can't do that, then again they're not worth keeping.